crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -1246,26 +1260,37 @@ impl From<&[u8]> for File {
     }
 }
 
-impl File {
-    pub fn new(meta: FileMeta, content: Vec<u8>) -> Self {
-        Self { meta, content }
+impl From<RopeReadRef> for File {
+    fn from(rope: RopeReadRef) -> Self {
+        File::from_rope(rope.clone_value())
     }
+}
 
-    pub fn meta(&self) -> &FileMeta {
-        &self.meta
+impl From<&Rope> for File {
+    fn from(rope: &Rope) -> Self {
+        File::from_rope(rope.clone())
```

Comment:
This feels redundant with `from_rope`. Maybe we should only have the `From` impl?